test_util/provide_aws.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
+import logging
+import os
+from abc import ABCMeta, abstractmethod
+
+import boto3
+import botocore
+import retrying
+
+
+LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
+logging.basicConfig(format=LOGGING_FORMAT, level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+AWS_VERIFY = True
+
+
+class AwsProvision(metaclass=ABCMeta):
```

Comment:
The intention was to split out the things that were to be used for arbitrary stacks in PR #622 and I couldn't figure out a good naming convention. How about
class CloudFormation-> DcosFormation(CloudFormation) & VpcFormation(CloudFormation) ?